test/tests/functional/pbs_test_tpp.py

```diff
@@ -825,6 +829,149 @@ def test_COMM_ROUTERS_with_nondefault_pbs_conf(self):
                           job=True, interactive=True, resv=True,
                           resv_job=True)
 
+    def start_gdb(self, comm_pid):
+        """
+        This function starts gdb and attach it to pbs_comm
+        :param comm_pid: Comm's Pid
+        :type comm_pid: Integer.
+        """
+        gdb_python = """
+import gdb
+gdb.execute("attach %s")
+o = gdb.execute("info threads", to_string=True)
+print(o)
+gdb.execute("detach")
```

Comment:
the next command you execute is quit, so "detach" should not be needed. This is only when we decide to use gdb.